samples/subsys/mgmt/osdp/sample.yaml

```diff
@@ -0,0 +1,13 @@
+sample:
+  description: OSDP Sample
+  name: osdp
+tests:
+  sample.subsys.mgmt.osdp:
```

Comment:
remove subsys

Reply:
Thanks! The naming scheme seems to vary across samples. Removed this here :)